packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -941,6 +942,16 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
         e.stopPropagation()
         return
       default:
+        const keyString = String.fromCharCode(keyCode)
+        if (/[a-zA-Z]/.test(keyString)) {
+          const highlightedIndex = this.getHighlightedIndexOnCharKeyPress(keyString)
+          if (highlightedIndex >= 0) {
+            this.setState({
```

Comment:
shouldn't we do `trySetState`? the user might still want to control `highlightedIndex` and opt out from this behavior

Reply:
good catch, thanks!